requests/adapters.py

```diff
@@ -408,7 +408,10 @@ def send(self, request, stream=False, timeout=None, verify=True, cert=None, prox
 
         chunked = not (request.body is None or 'Content-Length' in request.headers)
 
-        if isinstance(timeout, tuple):
+        
+        if isinstance(timeout, TimeoutSauce):
+            pass
```

Comment:
Can this be moved lower down the list? A timeout object isn't a tuple, so the original `isinstance(tuple)` check won't catch it, so let's try to avoid rearranging the code too much.